components/Dropdown.js

```diff
@@ -0,0 +1,9 @@
+
+export default function Dropdown({ sortOptions},{sortId}) {
+    return (
+        <select id={sortId}>
+        {sortOptions.map((s) => (
+           <option>{s}</option>
+        ))}</select>
```

Comment:
Just a code style thing, but you should run auto format to indent the inside of the select tag, and then also put the `</select>` on the following line. Generally put end tags on their own line so it allows the eye to easily scan through and see the contents of the element easier